node/core/dispute-coordinator/src/real/initialized.rs

```diff
@@ -832,6 +832,7 @@ impl Initialized {
 			.is_some();
 
 		// Update candidate votes.
+		let mut fresh_invalid_st_issuers = Vec::new(); // indexes of the validators issued 'invalid' statements
```

Comment:
nit: Avoid abbreviations whenever possible
nit: Move the dev comment to a separate line, two pane editor users on their laptop will thank you :)

Reply:
Thanks. I've fixed both.